### PR TITLE
feat: add StaticLegend

### LINKED ISSUE #21154: Static Legend: update swagger
Update swagger to include view options for Static Legend

`StaticLegend` is an object with the following properties
  - `colorizeRows`
  - `disable`
  - `heightRatio`
  - `layer`
  - `orientationThreshold`
  - `widthRatio`

As Static Legend is a work-in-progress, some additional properties may be added and some may be removed.

### PR DESCRIPTION
**Pull request by TCL735**
Closes #21154 

- Removes unused, conflicting, and confusing references to "Legend"
- Adds a collection of properties for the Static Legend which is a new feature for the UI

- [x] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass
- [x] http/swagger.yml updated (if modified Go structs or API)
- [x] Feature flagged (if modified API)
- [x] Documentation updated or issue created (provide link to issue/pr)  - _**documented in Giraffe**_
- [x] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

**Comment by TCL735** [comment]
force push was to rebase on master

**Comment by TCL735** [comment]
Ready for review.

**Comment by TCL735** [comment]
force push to resolve merge conflicts

**Comment by TCL735** [comment]
ready for review

File: http/swagger.yml [hunk 1]
```diff
@@ -8938,7 +8938,6 @@ components:
         - shape
         - axes
         - colors
-        - legend
```
Comment (danxmoran) [inline http/swagger.yml:8941]: `staticLegend` doesn't replace it as required?

**Review by danxmoran — APPROVED** [review]

File: http/swagger.yml (hunk 1 above)
Comment (TCL735) [reply http/swagger.yml:8941]: Correct. Static Legend is not required.

**Pull request merged**